i18n/en/docusaurus-plugin-content-docs/current/get-started/tutorial.mdx

```diff
@@ -0,0 +1,283 @@
+
+The note editor is a bit more complex because it includes elements that are compositions themselves, such as the buttons at the bottom. Still mostly the same appearance and data, but this time there’s a inline text editor between them.
+
+![Composition of a note editor](/img/tutorial/note-editor-composition-light.jpg#light-mode-only)
+![Composition of a note editor](/img/tutorial/note-editor-composition-dark.jpg#dark-mode-only)
+
+The Delete button is a feature, but it uses the same appearance as the back button. Note that the back button is not a feature because its purpose is not to deliver business value.
+
+### Purpose over essence
+
+In the diagrams above, every element is annotated with its purpose — whether it represents logic, data or UI. It makes for a convenient grouping of code by its technical purpose. In FSD, we maintain this grouping through segments. Segments are just folders inside of slices. Some common segment names are `ui/`, `model/`, and `api/`. 
+
+- `ui` contains everything that involves presentation — UI components, data formatting functions, hooks for UI logic, etc.
+- `model` contains the data and business logic — stores for certain types of objects, data mappers, hooks for business logic, etc.
+- `api` contains everything to do with communicating with external APIs — API calls, queries, mutations.
+
+This kind of grouping is beneficial because it allows you to predict where a certain module will be in the file system. If you’re looking for API query hooks, you don’t have to dig through the `hooks/` folder, ignoring all the UI and logic hooks.
+
+You’re not limited to these three segments, you could define your own, just make sure that they reflect the purpose of the code (the why), not the essence (the what).
+
+<p><em style={{ color: "indianred", fontWeight: "bold", fontStyle: "normal" }}>Bad</em> segment names: <code>components</code>, <code>hooks</code>, <code>helpers</code></p>
+
+<p><em style={{ color: "green", fontWeight: "bold", fontStyle: "normal" }}>Good</em> segment names: <code>assets</code>, <code>i18n</code>, <code>routes</code></p>
+
+With that in mind, a typical layer would look like this:
+
+```
+📂 features/
+  📂 edit-note/
+    📁 ui/
+    📁 api/
+    📄 index (a declaration of what other modules can import from here)
+  📁 …
+```
+
+### Business-less layers
+
+Talking about segments is a good segue into the two layers-exceptions: App and Shared. They are located on top and bottom of the layer stack respectively, and they differ structurally from other layers — they don’t have slices. The Shared layer doesn’t have slices because it’s not tied to any business domain, the App layer doesn’t have slices because it **is** the business domain.
+
+Usually, these two layers are the best places to define your custom segments. Don’t be afraid to create new segments, they don’t cost you anything. For example, your Shared layer could consist of the following segments:
+
+- `ui` — the UI kit
```

Comment:
nitpick: Probably should order them in alphabetical order, because they aren't layers, so people don't loose their shit trying to find extensions for ordering or renaming with `01_ui`

```
- api
- config
- i18n
- router
- ui
```
![image](https://github.com/feature-sliced/documentation/assets/74127809/90e45e0b-371e-486c-9b16-83b8dc955cfb)